### PR TITLE
Adding new STIS notebook Low_Count_Uncertainties.ipynb

### PR DESCRIPTION
**Pull request by Jackie-Brown**
Added STIS notebook Low_Count_Uncertainties.ipynb, requirements.txt, and pre-requirements.sh

### CODE REVIEW

**Comment by haticekaratay** [comment]
@Jackie-Brown ,
I ran this notebook locally and encountered the same error as reported on the CI.
image

image

**Comment by haticekaratay** [comment]
@Jackie-Brown , another important thing to note is that we are getting a deprecation warning for these two. The notebook will fail when new releases are out. 
```
UserWarning: GFIT is deprecated - stsci.tools v 3.4.12 is the last version to contain it.Use astropy.modeling instead.
  warnings.warn("GFIT is deprecated - stsci.tools v 3.4.12 is the last version to contain it."
UserWarning: NMPFIT is deprecated - stsci.tools v 3.5 is the last version to contain it.
  warnings.warn("NMPFIT is deprecated - stsci.tools v 3.5 is the last version to contain it.")
```

**Comment by sean-lockwood** [comment]
The notebook is calling `crds` in the Unix shell, but that package is not in the requirements file:

```python
!crds bestrefs --update-bestrefs --sync-references=1 --files ./mastDownload/HST/obgh07020/obgh07020_raw.fits
```

I believe the Unix call is silently failing because the `crds` executable cannot be found.

**Comment by Jackie-Brown** [comment]
@haticekaratay I took a look at the depreciation warning for NMPFIT and GFIT, and the notebook doesn't use either of these. Will the notebook still fail when the new version of stsci.tools is out? Otherwise the notebook looks ready to publish unless you have any other concerns

**Comment by haticekaratay** [comment]
> @haticekaratay I took a look at the depreciation warning for NMPFIT and GFIT, and the notebook doesn't use either of these. Will the notebook still fail when the new version of stsci.tools is out? Otherwise the notebook looks ready to publish unless you have any other concerns

@Jackie-Brown, I will take a look, as it's been a while, and submit my review here this week.

**Comment by sean-lockwood** [comment]
@Jackie-Brown @haticekaratay -
Those are known deprecation warnings when importing `stistools` for now.

File: notebooks/STIS/low_count_uncertainties/Low_Count_Uncertainties.ipynb [hunk 1]
```diff
@@ -0,0 +1,960 @@
+{
```
Comment (haticekaratay) [inline notebooks/STIS/low_count_uncertainties/Low_Count_Uncertainties.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.    raw_file = os.path.join('mastDownload', 'HST', target_id, f'{target_id}_tag.fits')
</pre><p>This raw file is also pointing to the same tag_file. Please check the previous comment and also this for the correct implementation. :)</p>

<pre class="ql-syntax" spellcheck="false">raw_file = os.path.join('mastDownload', 'HST', target_id, f'{target_id}_raw.fits')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Review by haticekaratay — APPROVED** [review]

**Pull request merged**